src/Worker.php

```diff
@@ -110,6 +111,11 @@ public function handle(Request $request, RequestContext $context): void
         } catch (Throwable $e) {
             $this->handleWorkerError($e, $sandbox, $request, $context, $responded);
         } finally {
+            $sandbox->flush();
```

Comment:
Then it's okay to remove other sandbox' forgetInstance call in FlushTemporaryContainerInstances?
(Not sure if it's necessary to call forgetInstance in FlushAuthenticationState)